tests/pyproject.toml

```diff
@@ -41,6 +41,9 @@ add-select = ["D212"]
 # D400 First line should end with a period, doesn't work when sentence spans 2 lines
 add-ignore = ["D105","D107","D203", "D205", "D400"]
 
+[tool.pytest.ini_options]
+asyncio_mode = "auto"
```

Comment:
It sounds like this mode means we don't need to decorate tests with any `@pytest.mark.asyncio` markers? If so that SGTM: let's get rid of them

Reply:
👍 I'm fine w/ either way